susemanager/src/mgr_bootstrap_data.py

```diff
@@ -232,40 +153,6 @@
     "venv-salt-minion", 
 ]
 
-AMAZONLINUX2 = [
```

Comment:
Amazon Linux required extra dependencies for `salt-minion`,  but the bundle does not require any of those packages, it seems.